config.go

```diff
@@ -579,3 +579,29 @@ func (b *Builder) SetDomainname(name string) {
 func (b *Builder) SetDefaultMountsFilePath(path string) {
 	b.DefaultMountsFilePath = path
 }
+
+// Comment returns the comment which will be set in the container and in
+//containers built using images buiilt from the container
```

Comment:
nit: buiilt to built ditto line 597

Reply:
it is a copy of existing text but can do

Reply:
fixed